src/components/views/View.js

```diff
@@ -33,7 +34,7 @@ const View = () => {
         }
       }
       default:
-        return <PlayView />;
+        return <AdminView />; // TODO To change between player and Teacher View
```

Comment:
Before merging, set the `AdminView` instead of the `CreateView`, and set back the default value to player.

Reply:
Yes sure, I will set back the default to player.

I also changed to `AdminView`, that was an oversight on my part.